Directory.Build.props

```diff
@@ -4,9 +4,16 @@
     <Authors>Microsoft.Toolkit</Authors>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <NoPackageAnalysis>true</NoPackageAnalysis>
-    <PackageIconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</PackageIconUrl>
-    <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/license.md</PackageLicenseUrl>
+  <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
+    <PropertyGroup>
+      <PackageLicenseFile>license.md</PackageLicenseFile>
+      <PackageIcon>nuget.png</PackageIcon>
+      <PackageIconUrl>nuget.png</PackageIconUrl>
+    </PropertyGroup>
```

Comment:
```suggestion
    <PackageIconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</PackageIconUrl>
    <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
    <PackageLicenseFile>license.md</PackageLicenseFile>
    <PackageIcon>nuget.png</PackageIcon>
```

The `PackageIconUrl` would still be the old one. We're also already in a property group here.

The `ItemGroup` below should be moved outside this Property Group down below.